src/constructs/vpc/vpc.ts

```diff
@@ -61,11 +61,11 @@ export class GuVpc extends Vpc {
       );
     }
 
-    node.setContext(`availability-zones:account=${account}:region=eu-west-1`, [
-      "eu-west-1a",
-      "eu-west-1b",
-      "eu-west-1c",
-    ]);
+    const contextKey = `availability-zones:account=${account}:region=eu-west-1`;
```

Comment:
Not for this PR necessarily, but I wonder why the region has been hard-coded here, rather than `this.region`?